packages/docs/docs/docs/deploying.md

```diff
@@ -115,10 +115,14 @@ And back in the original terminal:
 zos push --network local
 ```
 
-This creates a `zos.dev-<network_id>.json` file with all the information about your
-project in this specific network. You can read more about this file format
+This deploys `MyContract` to the specified network. If your project added other contracts (using the `add` command) they would be deployed as well. An important thing to understand is that the contracts deployed by the `push` command are implementation or logic contracts and are not intended to be used directly, rather to be linked to a proxy as we will see later in the [Upgrading your project](https://docs.zeppelinos.org/docs/upgrading.html) section.
```

Comment:
Question: are we talking about proxies at this stage, or do we refer to them as "instances"?

Reply:
@elopio :point_up: ?

Reply:
Yes, please let's talk about upgradeable instances and avoid proxy unless we are talking really low-level